### PR TITLE
[7.x] misc. SDET backports for week 01/01 - 07/01

### PR DESCRIPTION
**Pull request by theopnv**
### Purpose of this PR

- Backported [Enforce true isolation testing for upm ci](https://github.com/Unity-Technologies/Graphics/commit/e4b0ab13532adfb4e5c4d12b13d84aa3f8b3ddad)
- Backported [Update SDET tooling](https://github.com/Unity-Technologies/Graphics/pull/2992)
- Backported [Update bokken image for Linux tests](https://github.com/Unity-Technologies/Graphics/pull/2654)
- Backported [Yamato Use just python3 in cmd](https://github.com/Unity-Technologies/Graphics/pull/2865)
- Applied master changes to the PR template
- Bumped gfx-testframework to 7.8.0-preview

---
### Testing status
Enforce true isolation testing for upm ci:
-  https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/7.x.x%252Fsdet%252Fmisc-backports/.yamato%252F_projectcontext.yml%2523test_all_project_windows_2019.4/4793159/job
- https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/7.x.x%252Fsdet%252Fmisc-backports/.yamato%252F_projectcontext.yml%2523test_all_project_macos_2019.4/4793161/job

Update bokken image for Linux tests:
- https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/7.x.x%252Fsdet%252Fmisc-backports/.yamato%252Fhdrp-linux-vulkan.yml%2523HDRP_Linux_Vulkan_editmode_mono_Linear_2019.4/4848113
- https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/7.x.x%252Fsdet%252Fmisc-backports/.yamato%252Fshadergraph-linux-openglcore.yml%2523ShaderGraph_Linux_OpenGLCore_editmode_mono_Linear_2019.4/4848114

### CODE REVIEW

**Review by sophiaaar — APPROVED** [review]

**Comment by theo-pnv** [comment]
What changed for core/hdrp, for @JulienIgnace-Unity  & @sebastienlagarde :
- Explicitly added the following dependencies to the core package:
```
        "com.unity.ugui" : "1.0.0",
        "com.unity.modules.physics": "1.0.0"
```
- Explicitly added the following dependencies to the hdrp package:
```
        "com.unity.modules.video": "1.0.0",
        "com.unity.modules.animation": "1.0.0",
        "com.unity.modules.imageconversion": "1.0.0",
```

File: com.unity.render-pipelines.high-definition/package.json [hunk 1]
```diff
@@ -6,6 +6,9 @@
     "unityRelease": "14f1",
     "displayName": "High Definition RP",
     "dependencies": {
+        "com.unity.modules.video": "1.0.0",
+        "com.unity.modules.animation": "1.0.0",
```
Comment (sebastienlagarde) [inline com.unity.render-pipelines.high-definition/package.json:10]: Is it expected that we do this change? sanity check

Comment (sebastienlagarde) [reply com.unity.render-pipelines.high-definition/package.json:10]: I mean for 7.x. Want to be sure that those package above exist on 19.3

Comment (theo-pnv) [reply com.unity.render-pipelines.high-definition/package.json:10]: The upm tests passed https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/7.x.x%252Fsdet%252Fmisc-backports/.yamato%252F_projectcontext.yml%2523test_all_project_windows_2019.4/4793159/job so I'm guessing we're ok?

**Review by sebastienlagarde — APPROVED** [review]

**Review by JulienIgnace-Unity — APPROVED** [review]

**Pull request merged**